toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -890,6 +898,12 @@ The path of the script.
 This must be in the same directory or a sub-directory that the config file is located
 in.
 
+Mutually exclusive with `content`.
```

Comment:
Can we just / also say "Only one of path or content may be specified." to make it easier to read?

Reply:
Fixed.